include/sys/zio.h

```diff
@@ -138,12 +138,14 @@ enum zio_encrypt {
  */
 #define	ZIO_COMPRESS_LEGACY_ON_VALUE	ZIO_COMPRESS_LZJB
 #define	ZIO_COMPRESS_LZ4_ON_VALUE	ZIO_COMPRESS_LZ4
+#define	ZIO_COMPRESS_ZSTD_ON_VALUE	ZIO_COMPRESS_ZSTD
 
 #define	ZIO_COMPRESS_DEFAULT		ZIO_COMPRESS_OFF
 
 #define	BOOTFS_COMPRESS_VALID(compress)			\
 	((compress) == ZIO_COMPRESS_LZJB ||		\
 	(compress) == ZIO_COMPRESS_LZ4 ||		\
+	(compress) == ZIO_COMPRESS_ZSTD ||		\
```

Comment:
again, move new stuff to the end

Reply:
hat least before ON and OFF. this has been just merged from alan's code 1:1. will change it